sanic/server.py

```diff
@@ -404,11 +407,12 @@ def serve_multiple(server_settings, workers, stop_event=None):
     server_settings['host'] = None
     server_settings['port'] = None
 
-    if stop_event is None:
-        stop_event = Event()
+    def sig_handler(signal, frame):
+        log.info("Received signal {}. Shutting down.".format(
```

Comment:
If I remember correctly, last time I tried to do this it would raise an exception and spew a garbage exception in the logger when you tried to stop the server.

The specific commit for reference is ca9845988f24943dddc6b4351a3448f86a7402a3.

And then the re-adding commit is 4ca2e1d8e4dd21d9dcf2f120975bafdc13cd081d